packages/stores/src/mockStore.ts

```diff
@@ -316,7 +316,18 @@ const mock = (): TStores & { is_mock: boolean } => {
         portfolio: {
             active_positions: [],
             all_positions: [],
-            error: '',
+            error: {
```

Comment:
Do we need this object here? Isn't it just a string? I guess error is object in common store, not in portfolio store

Reply:
You are probably right. These changes came in when i pulled ur branch though. I didn't add this error object in mock

Reply:
Actually no. i think it needs to be defined where ever TCommonStoreError exists

Reply:
But error in portfolio store is string and TCommonStoreError is for error from common store, isn't it?

Reply:
no. They are both TCommonStoreError now. At least thats what i got from pulling ur branch

Reply:
@henry-deriv error in portfolio-store is always a string. not the same as in common-store. please revert your changes here. 🙏🏻 
<img width="920" alt="Screenshot 2023-07-05 at 10 05 41 AM" src="https://github.com/kate-deriv/deriv-app/assets/103177211/6728027d-524c-4f28-b3cb-82065c38e1ea">
<img width="920" alt="Screenshot 2023-07-05 at 10 06 32 AM" src="https://github.com/kate-deriv/deriv-app/assets/103177211/36adb46a-0ea3-4a77-b4a6-465c8ab662fc">
<img width="920" alt="Screenshot 2023-07-05 at 10 07 01 AM" src="https://github.com/kate-deriv/deriv-app/assets/103177211/cb744a60-b1e2-49ac-bd13-5ce0a25034f7">